### ISSUE TITLE
Engine 2 used fuel does not reset for second leg

### ISSUE THREAD

**Issue by mico975** [flybywiresim/a32nx#6301]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2021-12-02T01:37:49+00:00",
  "ref": "master",
  "sha": "8dc5bc79bbbd69f4a90c20816c0ea5e22575cbad",
  "actor": "tracernz",
  "event_name": "manual"
}
```

### Describe the bug

When doing multiple legs engine 2 used fuel doesn't reset on engine start for second leg. Engine 2 start instead resets engine 1 used fuel. Engine 1 used fuel reset works correctly.

### Expected behavior

Engine 2 to reset used fuel for engine 2.

### Steps to reproduce

1. Fly a normal flight.
2. Shutdown and prepare for next leg.
3. Start engine 2 normally and it resets used fuel for engine 1.

### References (optional)

![image](https://user-images.githubusercontent.com/78908672/144706994-f33fc981-7606-49a0-8315-26e4b133b41d.png)

### Additional info (optional)

I suspect it is just a single wrong simvar (FuelUsedLeft instead of FuelUsedRight) in the EngineControl file but I can't really confirm as I'm too dumb to build the thing myself.

### Discord Username (optional)

Mico#3145

**Comment by John-UK** [comment]
I checked the code in enginecontrol.h The code from issue #5486 is still the same. 
Engine 1 still works as expected. 
When engine 2 is restarted it resets the fuel used for engine 1. 

There are several different named vars used for fuel used now. It could that this is a display issue in the ECAM code or somewhere outside of enginecontrol.h the setFuelUsedRight is updated. 

I have not kept track of this function. It worked correctly at the time, but when it broke?

**Comment by mico975** [comment]
I just had a brief look at the code to see if I could find anything and this looked suspicious. I'm not familiar with the actual coding of anything so I guess I am wrong and someone more competent needs to look at this.

I believe it broke after #5895 got merged or somewhere around that time but could be wrong because I only noticed it after doing multiple legs shortly after.

**Comment by John-UK** [comment]
This appears to have occurred when the fuel reset was moved from engine state = 2 to engine state = 1. The attached pic shows that setFuelUsedLeft has been used twice for engine 1 and engine 2.
![image](https://user-images.githubusercontent.com/31592841/145728491-c5ef86e3-c774-4a22-8b8c-6ca678a312d8.png)

**Comment by Taz5150** [comment]
Thank you @John-UK for finding the problem with this issue. I have made a small fix in the code for PR#6284 which should be merged soon.
